numba_dpcomp/numba_dpcomp/mlir_compiler/lib/pipelines/lower_to_llvm.cpp

```diff
@@ -1755,6 +1755,43 @@ struct LowerReleaseContextOp
   }
 };
 
+struct PierUtilToLLVMPass
```

Comment:
Please add docs - at least to new entities - by adding doxygen comments describing high-level purpose and functionality as well as args/return types.

Reply:
done